scrapers/GammaEntertainment.yml

```diff
@@ -139,7 +139,7 @@ xPathScrapers:
               # if using the transport subdomain, parameters need to be passed
               # otherwise a cropped square image is returned by default
               - regex: (https:\/\/transform.+)
-                with: $1?width=960&height=543&enlarge=true
+                with: $1?width=1920&height=1080&enlarge=true
```

Comment:
I wasnt sure about the enlarge part, if it needs to be removed but all the sample images i found with transform urls seem to now be 1920x1080.
Merging as is and we can have another look later if needed